src/ast/legacy/try_catch_clause_processor.ts

```diff
@@ -18,6 +18,6 @@ export class LegacyTryCatchClauseProcessor extends LegacyNodeProcessor<TryCatchC
         const block = children.pop() as Block;
         const parameters = children.length ? (children.pop() as ParameterList) : undefined;
 
-        return [id, src, type, errorName, block, parameters, raw];
+        return [id, src, type, errorName, block, parameters, undefined, raw];
```

Comment:
I am curious how you attach a documentation to a **clause**. Is this how it works?

```contract Foo {
    uint x;
    function getX() public returns(uint) {
        return x;
    }
    
    function main() public {
        try this.getX() returns (uint) {
            
        } 
        /// Test
        catch {
            
        }
    }
}```

Or is it the case that clauses can't get a docstring, but we just include them for regularity?

Reply:
Looking at `latest_08.sourced.sol` it looks like docstring on clauses are ignored :). That answers my question.